tooling/src/hypothesistooling/__main__.py

```diff
@@ -67,127 +71,64 @@ def lint():
-        tools.update_for_pending_release()
+    print('Looks good to release!')
 
-    print('Building an sdist...')
+    tag_name = package.tag_name()
 
-    if os.path.exists(DIST):
-        shutil.rmtree(DIST)
+    print('Creating tag %s' % (tag_name,))
 
-    subprocess.check_output([
-        sys.executable, 'setup.py', 'sdist', '--dist-dir', DIST,
-    ])
+    tools.create_tag(tag_name)
 
-    if not on_master:
-        print('Not deploying due to not being on master')
-        sys.exit(0)
+    print('Uploading distribution')
+    package.upload_distribution()
 
-    if not has_release:
-        print('Not deploying due to no release')
-        sys.exit(0)
+    print('Pushing tag')
+    tools.push_tag(tag_name)
 
-    start_time = time()
-
-    prev_pending = None
-
-    # We time out after an hour, which is a stupidly long time and it should
-    # never actually take that long: A full Travis run only takes about 20-30
-    # minutes! This is really just here as a guard in case something goes
-    # wrong and we're not paying attention so as to not be too mean to Travis..
-    while time() <= start_time + 60 * 60:
-        jobs = tools.build_jobs()
-
-        failed_jobs = [
-            (k, v)
-            for k, vs in jobs.items()
-            if k not in PENDING_STATUS + ('passed',)
-            for v in vs
-        ]
-
-        if failed_jobs:
-            print('Failing this due to failure of jobs %s' % (
-                ', '.join('%s(%s)' % (s, j) for j, s in failed_jobs),
-            ))
-            sys.exit(1)
-        else:
-            pending = [j for s in PENDING_STATUS for j in jobs.get(s, ())]
-            try:
-                # This allows us to test the deploy job for a build locally.
-                pending.remove('deploy')
-            except ValueError:
-                pass
-            if pending:
-                still_pending = set(pending)
-                if prev_pending is None:
-                    print('Waiting for the following jobs to complete:')
-                    for p in sorted(still_pending):
-                        print(' * %s' % (p,))
-                    print()
-                else:
-                    completed = prev_pending - still_pending
-                    if completed:
-                        print('%s completed since last check.' % (
-                            ', '.join(sorted(completed)),))
-                prev_pending = still_pending
-                naptime = 10.0 * (2 + random.random())
-                print('Waiting %.2fs for %d more job%s to complete' % (
-                    naptime, len(pending), 's' if len(pending) > 1 else '',))
-                sleep(naptime)
-            else:
-                break
-    else:
-        print("We've been waiting for an hour. That seems bad. Failing now.")
-        sys.exit(1)
 
-    print('Looks good to release!')
+@task()
+def deploy():
+    print('Current head:', HEAD)
+    print('Current master:', MASTER)
```

Comment:
v minor: add spacing to `Current head:` so the hashes line up? Then a bit easier to see if they’re the same.